src/references/auth.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Authentication
+---
+
+## Authentication Sequence
+
+1. The User opens up a browser to check his cozy, using his *cozy app domain* (`user-home.mycozy.cloud/`, or `user-drive.mycozy.cloud/`, or ...)
+2. he's redirected to his *cozy user domain* (`user.mycozy.cloud`)
+3. The browser request the *cozy user domain*, which send back the authentication form. The user fills his password, sending it back to the  *cozy user domain*
+4. The server *cozy user domain* set an auth cookie for *cozy user domain* ONLY
+5. A redirect status response code (302) for the requested app domain (`user-home.mycozy.cloud/`, or `user-drive.mycozy.cloud/`, or ...) is sent to the browser
+5. The browser request the *cozy app domain* with a [jwt code](https://jwt.io/)
+6. The jwt code  is checked and decoded through the stack, which send back a bearer token in the request header to *cozy app domain*
+7. *cozy app domain* uses the bearer token to communicate with the stack through XHR request
+8. *cozy app domain* sends back the authenticated UI to the browser, and a new cookie for the *cozy app domain* (`user-home.mycozy.cloud/`, or `user-drive.mycozy.cloud/`, or ...)
+
+For each new app requested by the user, repeat steps
+(Steps 3 & 4 only happens when the user is not authenticated at all : meaning there is no auth cookie for *cozy user domain* )
+
+![Architecture](../img/dev/auth_sequence.png)
+
+
+## Authentication types
+
+### Password
+
+- Used by the user on the authentication form on the *cozy user domain* (`user.mycozy.cloud`)
+- It can be changed through the setting apps
+- It can be reset on the authentication form or in the setting app, by sending an email to the user
```

Comment:
A fact that may be relevant here: when the password is changed, all the old cookies are invalidated, and we recreate a new one for the current session. In practice, it means the user will need to login again in their other browsers.